src/App.tsx

```diff
@@ -1,7 +1,49 @@
 import React from 'react';
 
+type State = {
+  pressedKey: boolean;
+  pressedUserKey: string;
+};
+
+export class Click extends React.Component<{}, State> {
+  state: Readonly<State> = {
+    pressedKey: false,
+    pressedUserKey: '',
+  };
+
+  componentDidMount() {
+    document.addEventListener('keyup', this.handleKeyUp);
+  }
+
+  componentWillUnmount() {
+    document.removeEventListener('keyup', this.handleKeyUp);
+  }
+
+  handleKeyUp = (event: KeyboardEvent) => {
+    this.setState({
+      pressedKey: true,
+      pressedUserKey: event.key,
+    });
+  };
+
+  render() {
+    return (
+      <div>
+        <h1>Press any key</h1>
+        <p className="App__message">
+          {
+            this.state.pressedKey
+              ? `The last pressed key is [${this.state.pressedUserKey}]`
+              : 'Nothing was pressed yet'
```

Comment:
Empty string converts to false, so you can use just pressedUserKey

```suggestion
            this.state. pressedUserKey
              ? `The last pressed key is [${this.state.pressedUserKey}]`
              : 'Nothing was pressed yet'
```